store/useShapeDiverStorePlatform.ts

```diff
@@ -52,8 +52,12 @@ export const useShapeDiverStorePlatform =
 								baseUrl: platformUrl,
 							});
 							try {
+								const refreshToken =
+									localStorage.getItem("refresh_token");
 								const result =
-									await client.authorization.refreshToken();
+									await client.authorization.refreshToken(
```

Comment:
@snabela I see potential issue:
If we pass refreshToken to the authorization.refreshToken - the token from platform will be unvalidated.
So, we also have to set new one to the localStorage.